source/remoteClient/__init__.py

```diff
@@ -0,0 +1,22 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2015-2025 NV Access Limited, Christopher Toth, Tyler Spivey, Babbage B.V., David Sexton and others.
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+from .client import RemoteClient
+
+client: RemoteClient = None
```

Comment:
I now realize that client is also a submodule. Interestingly the linter doesn't seem to complain about this.
May be call this remoteClient as well to be sure that there's no name collision.
```suggestion
remoteClient: RemoteClient = None
```